src/main/java/pl/llp/aircasting/screens/sessions/shareSession/ShareSessionActivity.java

```diff
@@ -102,11 +87,36 @@ public void onClick(View view) {
                 mShareSessionView.showSensors();
                 break;
             case R.id.share_file:
-                shareFile();
+                mShareSessionView.showEmailField();
+                break;
+            case R.id.send_file:
+                sendFile();
                 break;
         }
     }
 
+    private static final String EXPORT_PATH = "/api/sessions/export_by_uuid.json";
+
+    private void sendFile() {
```

Comment:
You could consider moving this method to an object the deals with http calls (usually some driver in the app), so that the activity doesn't have to be concerned with that - ideally, it should just work as a controller and don't do too much by itself.